Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -9356,13 +9356,10 @@ WebXRLayersAPIEnabled:
   condition: ENABLE(WEBXR_LAYERS)
   defaultValue:
     WebKitLegacy:
-      "PLATFORM(VISION) && ENABLE(WEBXR_WEBGPU_BY_DEFAULT)": true
```

Comment:
without this change, XR Dinos is broken when `ENABLE_WEBXR_WEBGPU_BY_DEFAULT=1` because it tries to use XRWebGLBinding which we don't implement.

WebGPU + WebXR doesn't require full layers support, just a subset. We are not supposed to advertise the feature as enabled